src/App/Tracker/Controller/Hooks/ReceivePullsHook.php

```diff
@@ -61,11 +61,16 @@ protected function prepareResponse()
 		// If the item is already in the database, update it; else, insert it.
 		if ($this->checkIssueExists((int) $this->data->number))
 		{
-			$this->updateData();
+			$result = $this->updateData();
 		}
 		else
 		{
-			$this->insertData();
+			$result = $this->insertData();
+		}
+
+		if (!$result)
+		{
+			$this->getContainer()->get('app')->close();
```

Comment:
The parent controller deals with the close.  IMO this should be a different message, not just a blank return (look at the screen cap I attached earlier).